packages/bun-types/bun.d.ts

```diff
@@ -1265,7 +1265,7 @@ declare module "bun" {
   type ModuleFormat = "esm"; // later: "cjs", "iife"
 
   interface BuildConfig {
-    entrypoints: string[]; // list of file path
+    entrypoints: [string, ...string[]]; // list of file path
```

Comment:
this type change may break some uses of the api

insight from @Didas-git:

<img width="806" alt="image" src="https://github.com/oven-sh/bun/assets/24465214/dc09ac2b-a17e-4747-8fcf-651eddd0ca2a">

imo we shouldn't do this change.

Reply:
To explain what i meant in the last line:
When it comes to this tuple types tss will do a lot of instanciations wich is pretty much useless if you are not making use of the type to begin with.

_the white theme hurts me tho_

Reply:
makes sense, thanks for the review! i've reverted the type change